src/components/comment/ActionDropdown.scss

```diff
@@ -0,0 +1,3 @@
+.bdl-Comment-actionDropdown .icon-ellipsis {
```

Comment:
Where is `icon-ellipsis` used?

Reply:
It's the icon used in `MediaMenu` component.
I could theoretically extend `MediaMenu` with optional prop of icon size and modify it directly there but I'm not sure if that's worth doing for just one use case.

Reply:
Done